tests/10-migration-backend-tests/testcafe/migrate_contact_emails.spec.js

```diff
@@ -4,8 +4,8 @@ import { doMigration } from "./util.js";
 
 const adminUser = Role('https://islandora-idc.traefik.me/user/login', async t => {
     await t
-        .typeText('#edit-name', 'admin')
-        .typeText('#edit-pass', 'password')
+        .typeText('#edit-name', process.env.DRUPAL_DEFAULT_ACCOUNT_NAME)
+        .typeText('#edit-name', process.env.DRUPAL_DEFAULT_ACCOUNT_PASSWORD)
```

Comment:
same type on `#edit-name`

Reply:
There are more of these, but I didn't mark them -- all the files changed have `#edit-name` where it should be `#edit-pass`